src/index.js

```diff
@@ -278,10 +279,16 @@ module.exports = (
   }
 
   // Do we have an `Accept` header set up in the form, but it hasn't been added yet?
-  if (formData.header && formData.header.Accept && har.headers.find(hdr => hdr.name === 'Accept') === undefined) {
+  if (
+    formData.header &&
+    Object.keys(formData.header).find(h => h.toLowerCase() === 'accept') &&
+    !har.headers.find(hdr => hdr.name.toLowerCase() === 'accept')
+  ) {
+    const acceptHeader = Object.keys(formData.header).find(h => h.toLowerCase() === 'accept');
```

Comment:
Wanna pre-define `Object.keys(formData.header).find(h => h.toLowerCase() === 'accept')` into a variable somewhere so you're not writing it out three times?